src/test/kotlin/note/mothers/NoteMother.kt

```diff
@@ -28,4 +28,6 @@ object NoteMother {
     }
 
     fun getIdentifierFrom(note: Note) = note.id
+
+    fun getValidNote() = getValidNoteWithDescription()
```

Comment:
Why is this needed? A function that calls another function without any new parameter isn't a good sign.
In the test cases you can create a valid note with description and it will do the job